apps/worker/src/app/workflow/services/workflow-queue.service.ts

```diff
@@ -55,14 +55,6 @@ export class WorkflowQueueService extends QueueService<IJobData> {
     });
   }
 
-  public async gracefulShutdown() {
-    // Right now we only want this for testing purposes
-    if (process.env.NODE_ENV === 'test') {
-      await this.bullMqService.queue.drain();
-      await this.bullMqService.worker.close();
-    }
-  }
```

Comment:
Used just for testing. Now this will be implemented in the BullMqService that this service is dependant.